pulsar-client-tools/src/test/java/org/apache/pulsar/client/cli/TestCmdProduce.java

```diff
@@ -72,4 +78,26 @@ public void testBuildSchema() {
         assertEquals(SchemaType.JSON, composite2.getKeySchema().getSchemaInfo().getType());
         assertEquals(SchemaType.AVRO, composite2.getValueSchema().getSchemaInfo().getType());
     }
+
+    @Test
+    public void generateAvroMessageBodies() throws Exception {
```

Comment:
can you add an example for keyvalue ?

Reply:
This method handles only the value part.
The key part is not supported by this pr

Reply:
we're not testing that in KV case the value schema is passed correcty